Tasks/PackerBuildV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,7 +1,7 @@
 {
-  "loc.friendlyName": "Build Machine Image",
+  "loc.friendlyName": "JDG Build Machine Image",
   "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=845329)",
-  "loc.description": "Build machine image using Packer. This image can be used for Azure Virtual machine scale set deployment",
+  "loc.description": "JDG Build machine image using Packer. This image can be used for Azure Virtual machine scale set deployment",
```

Comment:
Make sure these changes are reverted

Reply:
forgot to stage a change, good catch, updated.